src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -46,6 +46,19 @@ public static function cron(string $expression, object $message): self
         return new self(CronExpressionTrigger::fromSpec($expression), $message);
     }
 
+    public static function hashedCron(string $expression, object $message, string $context = null): self
+    {
+        if (!$context && $message instanceof \Stringable) {
```

Comment:
I would remove the context in favor of forcing messages to implement Stringable.